src/cpu/mod.rs

```diff
@@ -841,6 +841,18 @@ impl Cpu {
                 self.set_reg_16(Register16Bit::SP, hl);
                 (pc + 1, 2)
             },
+            Instruction::LdSpPlusS8ToHl() => {
```

Comment:
We're now copying boilerplate multiple times.  Add a TODO to consolidate ALU operations .

Reply:
Also put this in its own function

Reply:
Still no TODO comment, please add dis

Reply:
oops sorry, imma add